operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -1422,3 +1419,41 @@ func (r *Cluster) GetDecommissionBrokerID() *int32 {
 func (r *Cluster) SetDecommissionBrokerID(id *int32) {
 	r.Status.DecommissioningNode = id
 }
+
+// getNodePoolsFromSpec returns the NodePools defined in the spec.
+// This contains the primary NodePool (driven by spec.replicas for example), and the
```

Comment:
```suggestion
// This contains the default NodePool (driven by spec.replicas for example), and the
```

Not sure what the terminology is but let's be consistent. If the primary pool has the name "default", let's rename the constant otherwise update this comment.

Reply:
Right, this is an oversight. default nodepool is the correct terminology. Updated the comment